MicrosoftTeams.schema.json

```diff
@@ -805,6 +805,24 @@
       },
       "description": "When a group install scope is selected, this will define the default capability when the user installs the app",
       "additionalProperties": false
+    },
+    "configurableProperties": {
```

Comment:
@Niranjan-S seems we have to include these changes into `Microsoft.Teams.Apps.Schema` project as well

Reply:
@greybax Why we need these changes?